jetty-client/src/main/java/org/eclipse/jetty/client/HttpClient.java

```diff
@@ -1160,9 +1160,11 @@ static boolean isSchemeSecure(String scheme)
         return HttpScheme.HTTPS.is(scheme) || HttpScheme.WSS.is(scheme);
     }
 
-    protected ClientConnectionFactory newSslClientConnectionFactory(ClientConnectionFactory connectionFactory)
+    protected ClientConnectionFactory newSslClientConnectionFactory(SslContextFactory sslContextFactory, ClientConnectionFactory connectionFactory)
```

Comment:
Is this a breaking API change?  anything we can do to keep the original signature as a deprecated on in case some have extended it?